app/src/main/java/com/codesoom/domain/ProductRepository.java

```diff
@@ -0,0 +1,89 @@
+package com.codesoom.domain;
+
+import com.codesoom.exception.ProductNotFoundException;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * 상품 데이터를 관리합니다.
+ * 추상화한 상품 데이터 접근 기능을 제공합니다.
```

Comment:
두 줄 중 첫번째 줄만 남겨도 괜찮겠습니다.
```suggestion
 * 상품 데이터를 관리합니다.
```